composer.json

```diff
@@ -12,15 +12,14 @@
 	],
 	"minimum-stability": "stable",
 	"require": {
-		"php": ">=7.0",
+		"php": ">=7.3 | >=8.0",
```

Comment:
Is this | because we need to support both PHP 7.4 and PHP 8.2 valueframe for a while?

Reply:
Bumped in a later commit to just `>= 8.1`

Reply:
Yeah just noticed, good job 👍